nifi-frontend/src/main/frontend/.eslintrc.json

```diff
@@ -26,7 +26,8 @@
                     {
                         "suffixes": [""]
                     }
-                ]
+                ],
+                "@angular-eslint/prefer-standalone": "warn"
```

Comment:
Do we want to just turn this rule off?

Reply:
I considered turning it off but decided to make it a warning as it is something we should strive to resolve going forward. We should make all components standalone, these warning remind us of which ones still need converted.